javascript/index.js

```diff
@@ -0,0 +1,87 @@
+function Book(bookTitle, bookAuthor) {
+  this.bookTitle = bookTitle;
+  this.bookAuthor = bookAuthor;
+}
+
+function DisplayBook() {
+  this.books = [];
+}
+
+DisplayBook.prototype.addBook = function (book) {
+  this.books.push(book);
+  this.displayBook(book);
+  this.saveBooksToLocalStorage();
+};
+
+DisplayBook.prototype.displayBook = function (book) {
+  const bookList = document.getElementById('bookList');
+  const div = document.createElement('div');
+  div.classList.add('book');
+
+  const title = document.createElement('div');
+  title.textContent = `Title: ${book.bookTitle}`;
+  div.appendChild(title);
+
+  const author = document.createElement('div');
+  author.textContent = `Author: ${book.bookAuthor}`;
+  div.appendChild(author);
+
+  const deleteBtn = document.createElement('button');
+  deleteBtn.textContent = 'Delete';
+  deleteBtn.classList.add('delete-btn');
+  div.appendChild(deleteBtn);
+
+  bookList.appendChild(div);
+};
+
+DisplayBook.prototype.clearForm = function () {
+  document.getElementById('title').value = '';
+  document.getElementById('author').value = '';
+};
+
+DisplayBook.prototype.deleteBook = function (bookTitle, bookAuthor) {
+  // eslint-disable-next-line max-len
+  this.books = this.books.filter((book) => !(book.bookTitle === bookTitle && book.bookAuthor === bookAuthor));
+  this.saveBooksToLocalStorage();
+};
+
+DisplayBook.prototype.saveBooksToLocalStorage = function () {
+  localStorage.setItem('books', JSON.stringify(this.books));
+};
```

Comment:
- [ ] Your local storage cannot keep more that one book. every time a book is added, the old one is deleted. Its my suggestion that you add the book at the books array and then you save the array to local storage. 😓 
![redo](https://github.com/Obote/AwesomeBooks/assets/99656687/376ed8fd-cbdf-4bc1-a5cb-980343a6e0b6)

- [ OPTIONAL] What do you think about using function programming to implement this. 🤔